src/overview/utils/helpers/OverviewUserSettings.ts

```diff
@@ -9,26 +9,29 @@ type OverviewUserSettings = {
 };
 
 type WelcomeSettings = {
-  hideWelcome: boolean;
+  hideWelcome?: boolean;
   save: (showWelcome: boolean) => void;
   clear: () => void;
 };
 
-const parseOrClean = (key) => {
+// eslint-disable-next-line @typescript-eslint/no-unnecessary-type-parameters
```

Comment:
?

Reply:
but its not working the generic here or something else as u do 
`parseOrClean<WelcomeSettings>(key) as WelcomeSettings;`

Reply:
oh that was a mistake I am cleaning this up